src/js/Router.js

```diff
@@ -1,15 +1,15 @@
-import { parse, stringify } from 'qs';
+import { stringify } from 'qs';
 import Route from './Route';
 
 /**
  * A collection of Laravel routes. This class constitutes Ziggy's main API.
  */
 export default class Router extends String {
     /**
-     * @param {String} name - Route name.
-     * @param {(String|Number|Array|Object)} params - Route parameters.
-     * @param {Boolean} absolute - Whether to include the URL origin.
-     * @param {Object} config - Ziggy configuration.
+     * @param {String} [name] - Route name.
```

Comment:
Just a jsdoc change for IDE to understand the params are all optional

Reply:
Nice, didn't know about this. Thanks.